schemas/item-schema.json

```diff
@@ -112,6 +112,7 @@
       "title": "ItemType",
       "description": "An enumeration.",
       "enum": [
+        "Booster",
```

Comment:
We don't really need this new item type. Can you make it work without it?

Reply:
If you say so, I'll find another way.